### PR TITLE
Print out errors for timestepping example

### PR DESCRIPTION
**Pull request by abarret**
Currently shows second order accuracy for manufactured solution.

### CODE REVIEW

File: INSVCTwoFluidStaggeredHierarchyIntegrator.cpp [hunk 1]
```diff
@@ -456,6 +480,7 @@ INSVCTwoFluidStaggeredHierarchyIntegrator::integrateHierarchy(const double curre
                 double thn_lower = 0.5 * ((*thn_data)(idx_c_low) + (*thn_data)(idx_c_up)); // thn(i-1/2,j)
                 (*f_un_data)(idx) = (*f_un_data)(idx) + C * thn_lower * (*un_data)(idx);
                 (*f_us_data)(idx) = (*f_us_data)(idx) + C * convertToThs(thn_lower) * (*us_data)(idx);
+                if (idx(0) == 0 && idx(1) == 0) pout << "(*us_data)(" << idx << ") = " << (*us_data)(idx) << "\n";
```
Comment (bindi-nagda) [inline INSVCTwoFluidStaggeredHierarchyIntegrator.cpp:483]: I assume this was being used during debugging but may not be needed anymore

File: time_stepping.cpp [hunk 2]
```diff
@@ -123,6 +123,14 @@ main(int argc, char* argv[])
...
+        if (IBTK::abs_equal_eps(loop_time, next_viz_dump_time, 0.1 * dt) || loop_time >= next_viz_dump_time)
+        {
+            pout << "\nWriting visualization files...\n\n";
+            ins_integrator->setupPlotData();
+            visit_data_writer->writePlotData(patch_hierarchy, iteration_num, loop_time);
+            next_viz_dump_time += viz_dump_time_interval;
+        }
         // Main time step loop
         while (!IBTK::rel_equal_eps(loop_time, time_end) && ins_integrator->stepsRemaining())
         {
```
Comment (bindi-nagda) [inline time_stepping.cpp:136]: Btw, I should've mentioned this during the last pr, but I was running into an error on my machine when setupPlotData() was being called. The error was saying the number of arguments passed to hier_math_ops.interp was more than expected. When I made the follow modification (i.e. interpolating side centered data to cell centers) inside the setupPlotDataSpecialized() routine in INSVCTwoFluidStaggeredHierarchyIntegrator:

```
hier_math_ops.interp(un_draw_idx,  d_un_draw_var, un_idx,  d_un_sc_var,  nullptr, d_integrator_time,                        synch_cf_interface);
hier_math_ops.interp(us_draw_idx,  d_us_draw_var, us_idx,  d_us_sc_var,  nullptr, d_integrator_time,                        synch_cf_interface);
```

where d_un_draw_var and d_us_draw_var were changed to a cell variables from node variables, the error was resolved. Not sure if the initial error is due to me having a different build of IBAMR.

File: INSVCTwoFluidStaggeredHierarchyIntegrator.cpp (hunk 1 above)
Comment (abarret) [inline INSVCTwoFluidStaggeredHierarchyIntegrator.cpp:483]:
```suggestion

```

File: time_stepping.cpp (hunk 2 above)
Comment (abarret) [reply time_stepping.cpp:136]: You are using an out-of-date version of IBAMR. This was fixed in PR #1493 of IBAMR, which was merged November 15, 2022.

**Pull request merged**